python/ray/_private/ray_logging/constants.py

```diff
@@ -45,3 +45,4 @@ class LogKey(str, Enum):
     FILENAME = "filename"
     LINENO = "lineno"
     EXC_TEXT = "exc_text"
+    CREATED = "created"
```

Comment:
we need to call it `timestamp_ns` as this is what the log viewer recognizes and we can use `time.time_ns()` to get the value.